plotly/tests/test_optional/test_figure_factory/test_figure_factory.py

```diff
@@ -442,9 +433,9 @@ def test_simple_streamline(self):
             'type': 'scatter',
             'mode': 'lines'
         }
-        self.assertListEqual(strln['data'][0]['y'][0:100],
-                             expected_strln_0_100['y'])
-        self.assertListEqual(strln['data'][0]['x'][0:100],
+        self.assertListEqual(list(strln['data'][0]['y'][0:100]),
```

Comment:
🚨 Breaking Change

What is it now? A numpy array?

Reply:
Yes